packages/eslint-plugin-template/src/rules/accessibility-valid-aria.ts

```diff
@@ -1,13 +1,29 @@
-import { aria } from 'aria-query';
-import { TmplAstBoundAttribute, TmplAstTextAttribute } from '@angular/compiler';
+import { aria, ARIAProperty, ARIAPropertyDefinition } from 'aria-query';
+import {
+  AST,
+  ASTWithSource,
+  LiteralArray,
+  LiteralMap,
+  LiteralPrimitive,
+  TmplAstBoundAttribute,
+  TmplAstTextAttribute,
+} from '@angular/compiler';
 
 import {
   createESLintRule,
   getTemplateParserServices,
 } from '../utils/create-eslint-rule';
 
+// As in the time it being write the `ARIAPropertyDefinition` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/64ae44025dbd41afb3b865ca37deddef7de1c5fb/types/aria-query/index.d.ts#L302-L306)
+// is incorrect (`value` should be `values`), we created this hackish type to avoid `any`.
+type ARIAPropertyDefinitionHack = Omit<ARIAPropertyDefinition, 'value'> & {
+  readonly values: ARIAPropertyDefinition['value'];
+};
```

Comment:
I hope the comment is clear enough to express the intent.

Reply:
Thanks yes, made sense, just added some minor English language suggestions

Reply:
I've opened a PR in the DefinitelyTyped repo: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50528. What do you think about delaying this merge a little bit until the PR is merged? Of course, if it takes too long to merge, we can move on as is.

Reply:
I removed this as https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50528 has been merged.